open-subshare/REPWRITERSPECS/BANNO.NEWSUBCREATE.V1.POW

```diff
@@ -546,34 +764,77 @@ PRINT TITLE="BANNO.NEWSUBCREATE.V1.POW"
        NEWLINE
        PRINT "   "+Q+"canOpenSubShare"+Q+": true,"
        NEWLINE
+
+[* RESULTS START
+*]
        PRINT "  "+Q+"results"+Q+": {"
        NEWLINE
-
+[* if new names are not allowed or if they are allowed but the max new names
+** are nset to 0 or and there are no eligible copy names then set new names
+** allowed flag to 0
+*]
        PRINT "    "+Q+"canAddNames"+Q+": "
-       IF VALUE(SACNAMEALLOWFLAG)=0 THEN
+       IF VALUE(SACNAMEALLOWFLAG)=0 OR
+         (VALUE(SACNAMEALLOWFLAG)=1 AND
+          SACNAMEMAXNEW=0 AND
+          COPYNAMELISTCOUNT=0) THEN
```

Comment:
Hi Tom, Won't COPYNAMELISTCOUNT still be 0 at this point in the program as BUILDCOPYNAMELIST hasn't been called yet, so the condition will fail in the RGSTATE NAMEPRELOAD? 
Or will Procedure BUILDCOPYNAMELIST, have already been called at this point in RGSTATE CREATESHARE? I can't tell looking at the PO code, as the UX code calls that.